src/main.py

```diff
@@ -25,6 +25,12 @@ def send_video_event(sid, data):
     sio.emit(event="receive_video_event", data=data['eventInfo'], room=data['roomId'], skip_sid=sid)
     return "OK"
 
+@sio.event
+def send_message(sid, data):
+    print(sid, data)
+    sio.emit(event="receive_message", data=data['message'], room=data['roomId'], skip_sid=sid)
```

Comment:
We should also send sid so that we can display author of the message in client. I think emit method should look like this:
`
    sio.emit(
        event = "receive_message", 
        data = {
            'from': sid,
            'content' : data['message']
        }, 
        room = data['roomId'], 
        skip_sid = sid,
    )
`